include/boost/json/object.hpp

```diff
@@ -1166,6 +1166,51 @@ class object
     std::size_t
     erase(string_view key) noexcept;
 
+    /** Erase an element preserving order
+
+        Remove the element pointed to by `pos`, which must
+        be valid and dereferenceable. Thus the @ref end()
+        iterator (which is valid but cannot be dereferenced)
+        cannot be used as a value for `pos`.
+        All references and iterators are invalidated.
```

Comment:
Is this true? Or does it only invalidate iterators from pos and later?

Reply:
However for the key erase function I don't see a reason to be more specific since we are not passing in an iterator.

Reply:
agreed